File: c/1012.c
#include <stdio.h>
#define PI 3.14159

int main() {

    double a1, b1, c1;
    float triangulo, circulo, trapezio, quadrado, retangulo;
   
    scanf("%lf %lf %lf", &a1, &b1, &c1);
    
    triangulo = (a1 * c1) / 2;
    circulo = PI * pow(c1, 2);
    trapezio = ((a1 + b1) * c1) / 2;
    quadrado = b1 * b1;
    retangulo = a1 * b1;
   
    printf("TRIANGULO: %.3f\n", triangulo);
    printf("CIRCULO: %.3f\n", circulo);
    printf("TRAPEZIO: %.3f\n", trapezio);
    printf("QUADRADO: %.3f\n", quadrado);
    printf("RETANGULO: %.3f\n", retangulo);
  
    return 0;
}

File: python/1012.py
n = input()


(a, b, c) = n.split()

pi = 3.14159
area_triangulo = float(a) * float(c) / 2
area_circulo = pi * float(c) ** 2
area_trapezio = ((float(a) + float(b)) * float(c)) / 2
area_quadrado = float(b) ** 2
area_retangulo = float(a) * float(b)


print("TRIANGULO: %.3f" % area_triangulo)
print("CIRCULO: %.3f" % area_circulo)
print("TRAPEZIO: %.3f" % area_trapezio)
print("QUADRADO: %.3f" % area_quadrado)
print("RETANGULO: %.3f" % area_retangulo)
